src/main/java/dao/impl/CartDaoImpl.java

```diff
@@ -0,0 +1,89 @@
+package dao.impl;
+
+import dao.CartDao;
+import model.Cart;
+import model.Product;
+import model.User;
+import org.apache.log4j.Logger;
+import util.DBConnection;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.Optional;
+
+public class CartDaoImpl implements CartDao {
+
+    private static final Logger logger = Logger.getLogger(UserDaoImpl.class);
+
+    @Override
+    public void addCart(Cart value) {
+
+        String sql = String.format("INSERT INTO cart(idUser) VALUES('%s')",
```

Comment:
what about prepared statement in all methods?
try to move all queries in class

Reply:
> what about prepared statement in all methods?
> try to move all queries in class

this is the next task and it isn't required